React/Base/RCTTVRemoteHandler.m

```diff
@@ -32,6 +32,9 @@
 NSString *const RCTTVRemoteEventUp = @"up";
 NSString *const RCTTVRemoteEventDown = @"down";
 
+NSString *const RCTTVRemoteEventPageUp = @"pageUp";
```

Comment:
Could you please also add these as `extern` lines in `React/Base/RCTTVRemoteHandler.h`?